doc/1/controllers/document/m-update/index.md

```diff
@@ -0,0 +1,156 @@
+---
+code: true
+type: page
+title: mUpdate
+description: Updates multiple documents
+---
+
+# mUpdate
+
+Updates multiple documents.
+
+---
+
+:::: tabs
+::: tab Java
+
+## Arguments
+
+```java
+public CompletableFuture<ConcurrentHashMap<String, ArrayList<Object>>> mUpdate(
+      final String index,
+      final String collection,
+      final ArrayList<ConcurrentHashMap<String, Object>> documents)
+throws NotConnectedException, InternalException
+
+public CompletableFuture<ConcurrentHashMap<String, ArrayList<Object>>> mUpdate(
+      final String index,
+      final String collection,
+      final ArrayList<ConcurrentHashMap<String, Object>> documents,
+      final UpdateOptions options)
+throws NotConnectedException, InternalException
+```
+
+| Arguments          | Type                                                    | Description                       |
+| ------------------ | ------------------------------------------------------- | --------------------------------- |
+| `index`            | <pre>String</pre>                                       | Index                             |
+| `collection`       | <pre>String</pre>                                       | Collection                        |
+| `documents`        | <pre>ArrayList<ConcurrentHashMap<String, Object>></pre> | ArrayList containing the documents to update |
+| `options`          | <pre>UpdateOptions</pre><br>(`null`)                    | Query options |
```

Comment:
```suggestion
| `retryOnConflict`  | <pre>Integer</pre> (optional)                | The number of times the database layer should retry in case of version conflict |
| `waitForRefresh`   | <pre>Boolean</pre> (optional)                | If set to `true`, Kuzzle will wait for the persistence layer to finish indexing |
```